silabs-multiprotocol/rootfs.amd64/usr/local/bin/zigbeed

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+echo "Starting zigbeed with aarch64 emulation..."
```

Comment:
please use bashio

Reply:
I guess you will only use it as wrapper? Mabe have it then only as comment?

Reply:
Yeah its only mean to be a wrapper. I make it a comment.